content/source/docs/enterprise/run/index.html.md

```diff
@@ -82,6 +82,27 @@ If you need to temporarily stop runs from being queued, you can lock the workspa
 
 You can find the lock button in the workspace settings page. Locking a workspace requires write or admin access.
 
+## Installing Terraform Providers
+
+### Providers Distributed by HashiCorp
+
+TFE runs `terraform init` before every plan or apply, which automatically downloads any [providers](/docs/configuration/providers.html) Terraform needs.
+
+Private installs of TFE can automatically install providers as long as they can access registry.terraform.io. If that isn't feasible due to security requirements, you can manually install providers. Use [the `terraform-bundle` tool][bundle] to pre-download all the providers you need and bundle them with a version of Terraform, use the admin pages to upload the bundle as a new Terraform version, and configure your workspaces to use that bundled Terraform version.
```

Comment:
I might shorten the last sentence here, since the tool documentation covers those steps, and just say "Use the `terraform-bundle` tool to provide a custom version of Terraform that includes the necessary providers."